bin/_run-docker

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+export USE_VIRTUAL_HOSTS=true
+
+export PATH="/app/node_modules/.bin:$PATH"
+
+cd /app && node -e 'require("babel-polyfill"); require("./server/server");'
```

Comment:
Isn't `WORKDIR` already set to `/app`?

Reply:
We should probably do `exec node`, so node replaces the shell.

Reply:
I pulled this bin/_run-docker thing from screenshots, where it's been bitrotting for a long time. I think `CMD ["npm", "start"]` will probably be better. And it means we don't have to mess with adding bash to alpine.